src/staticsampler.jl

```diff
@@ -37,15 +37,15 @@ Static nested sampler with `nactive` active points and `ndims` parameters.
 
 `bounds` declares the Type of [`Bounds.AbstractBoundingSpace`](@ref) to use in the prior volume. The available bounds are described by [`Bounds`](@ref). `proposal` declares the algorithm used for proposing new points. The available proposals are described in [`Proposals`](@ref). If `proposal` is `:auto`, will choose the proposal based on `ndims`
 * `ndims < 10` - [`Proposals.Uniform`](@ref)
-* `ndims ≥ 10` - [`Proposals.RWalk`](@ref)
+* `ndims >= 10` - [`Proposals.RWalk`](@ref) or [`Proposals.RStagger`](@ref)
```

Comment:
Like I said previously, there is no "or" here: if `ndims ≥ 10` it will be `Proposals.RWalk`.

```suggestion
* `ndims ≥ 10` - [`Proposals.RWalk`](@ref)
```